### PR TITLE
Exposed getStakeRequestTypedData from utils

### LINKED ISSUE #120: Expose getStakeRequestTypedData from contract interacts/utils.
Currently `getStakeRequestTypedData` is exposed with Stake Helper which will be soon deprecated. 
Expose `getStakeRequestTypedData` from interacts/utils to support it for higher versions.

Stake helper should keep a proxy for compatibility.

### PR DESCRIPTION
**Pull request by sarvesh-ost**
Fixes #120 

Exposed `getStakeRequestTypedData` from utils.

### CODE REVIEW

**Comment by abhayks1** [comment]
Reviewing this ticket!

File: utils/Utils.js [hunk 1]
```diff
@@ -1,5 +1,20 @@
...
+ *
+ * @property {string} messageHash hash which is signed.
+ * @property {string} v v of signature generated by signing message hash.
+ * @property {string} r r of signature generated by signing message hash.
+ * @property {string} s s of signature generated by signing message hash.
+ * @property {string} signature signature generated by signing
+ *                              message hash represented into single string.
+ *
+ */
+
```
Comment (abhayks1) [inline utils/Utils.js:12]: Why this documentation is at the top?
Should it go with getStakeRequestTypedData method?

File: utils/Utils.js [hunk 2]
```diff
@@ -91,6 +106,53 @@ class Utils {
...
+  /**
+   * Returns EIP712TypedData StakeRequest Object.
+   *
+   * @param stakeAmountInWei Stake amount in wei.
+   * @param btStakeRequestNonce BrandedToken StakeRequest nonce.
+   * @param brandedTokenAddress BrandedToken contract address.
+   * @param gatewayComposerAddress Gateway composer contract address.
+   *
+   * @returns {EIP712TypedData} EIP712TypedData instance of StakeRequest.
+   * @returns {EIP712Signature} Signature generated after EIP712 signing.
```
Comment (abhayks1) [inline utils/Utils.js]: The function returns EIP712TypedData and not EIP712Signature.
EIP712Signature is generated when doing Account signing.

File: utils/Utils.js [hunk 3]
```diff
@@ -91,6 +106,53 @@ class Utils {
...
     );
   }
+
+  /**
+   * Returns EIP712TypedData StakeRequest Object.
+   *
+   * @param stakeAmountInWei Stake amount in wei.
+   * @param btStakeRequestNonce BrandedToken StakeRequest nonce.
+   * @param brandedTokenAddress BrandedToken contract address.
+   * @param gatewayComposerAddress Gateway composer contract address.
```
Comment (abhayks1) [inline utils/Utils.js:115]: Ordering of documentation mismatch with function arguments.

File: lib/helpers/stake/gateway_composer/StakeHelper.js [hunk 4]
```diff
@@ -229,51 +226,21 @@ class StakeHelper {
...
    * @returns {EIP712TypedData} EIP712TypedData instance of StakeRequest
-   * @returns {Object} Signature format:
-   *                  {
-   *                    messageHash: signHash,
-   *                    v: vrs[0],
-   *                    r: vrs[1],
-   *                    s: vrs[2],
-   *                    signature: signature
-   *                  }
+   * @returns {EIP712Signature} Signature generated after EIP712 signing.
```
Comment (abhayks1) [inline lib/helpers/stake/gateway_composer/StakeHelper.js]: This return is not needed. It was incorrect previously also.

File: utils/Utils.js (hunk 1 above)
Comment (0xsarvesh) [reply utils/Utils.js:12]: 1. `typedef` is usually kept at the top of the file for the reader to be known in priorly. 
2. Single typedef can be used at multiple places, in that case, it won't be possible to localize it with the method. 
3. This is also consistent with other files and repos.

**Pull request merged**